netmaster/master/netmaster.go

```diff
@@ -36,6 +36,13 @@ const (
 	defaultSkyDNSImage  = "skynetservices/skydns:latest"
 )
 
+// Run Time config of netmaster
+type nmRunTimeConf struct {
+	clusterMode string
+}
+
+var masterRTCfg nmRunTimeConf
```

Comment:
Should this type of information be added in global runtime structure? This allows for netmaster restartability - @shaleman WDYT?